src/handle.rs

```diff
@@ -45,15 +45,19 @@ pub fn find_next(
 
 pub async fn collect_data(info: Info) -> anyhow::Result<Data> {
     let mut clients = Clients::get_async().await?
+        .iter()
         .filter(|c| c.workspace.id != -1) // ignore nonexistent clients
         .filter(|w| !info.hide_special_workspaces || !w.workspace.id < 0)
+        .cloned()
```

Comment:
I think if u used `.into_iter()`, you could omit the `.cloned()`, but I'm not 100% sure

Reply:
Builds fine with ``.into_iter()``.